src/components/ComputeModal.js

```diff
@@ -101,6 +93,34 @@ const DataprocDiskSelector = ({ value, onChange }) => {
   ])
 }
 
+const SparkInterface = ({ interfaceName, synopsis, namespace, name, onDismiss }) => {
+  const interfaceDisplayName = Utils.switchCase(interfaceName,
```

Comment:
Why does it display this `logged in as dr. who` in the top right of the proxied iframe itself? Is this some backend configuration that can be changed?

![image](https://user-images.githubusercontent.com/6465084/135641859-69944ef1-bad5-4a9c-be7e-dc29369cf313.png)

Reply:
I think @rtitle kept it because it sounded cool 😛 

Joke aside, I see a comment on it on the [related section](https://docs.google.com/document/d/1Zr6qzICStrrJwYkfoIE7fTf9Ziu3f6JmhdA9Ao3Y8Bo/edit#heading=h.93ivn47x7ed) of the design doc. Was there any follow-up on it?

Reply:
Haha.. yeah I think the UIs display that if there is no auth. I did not look into how to change/remove it. Perhaps we could make a follow-on ticket.